### PR TITLE
Fix webhook hash bug

### PR DESCRIPTION
**Pull request by yashshah**

### CODE REVIEW

File: src/actions/webhook.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-var murmur = require('murmur')
+var murmur = require('murmurhash3js');
+var stringify = require('json-stable-stringify');
```
Comment (siddharthlatest) [inline src/actions/webhook.js]: we don't use semi-colons in this lib

File: src/actions/webhook.js [hunk 2]
```diff
@@ -53,7 +54,7 @@ var addWebhookService = function addWebhook(client, args, webhook) {
 
 	this.populateBody()
 
-	var hash = murmur.hash128(JSON.stringify(this.query)).hex()
+	var hash = murmur.x86.hash128(stringify(this.query));
```
Comment (siddharthlatest) [inline src/actions/webhook.js]: This murmur implementation seems x86 architecture specific. Any particular reason to use this particular lib?

Comment (yashshah) [inline src/actions/webhook.js]: murmur package was the main reason why there was bug. It was returning the same hash for different object. Changing it to any other hashing library will solve this problem.There is x64 architecture support as well.

File: src/actions/webhook.js (hunk 1 above)
Comment (yashshah) [inline src/actions/webhook.js]: Noted

**Pull request closed without merging**